src/train/training.rs

```diff
@@ -781,10 +780,12 @@ where
                 spec1
                     .adjusted_fitness
                     .partial_cmp(&spec2.adjusted_fitness)
-                    .expect(&*format!(
-                        "First: {}, second: {}, variance {}",
-                        spec1.adjusted_fitness, spec2.adjusted_fitness, variance
-                    ))
+                    .unwrap_or_else(|| {
+                        panic!(
+                            "First: {}, second: {}, variance {}",
+                            spec1.adjusted_fitness, spec2.adjusted_fitness, variance
+                        )
```

Comment:
shouldn't it be better to use safe code?

Reply:
It is safe